YunoHost/yunohost
# Update spectre meltdown checker

## The problem

Seems like there was a lot of activity on the spectre/meltdown checker repo so this needs update

## Solution

Ran : `git subtree pull --prefix src/yunohost/vendor/spectre-meltdown-checker https://github.com/speed47/spectre-meltdown-checker/ v0.37 --squash`

## PR Status

Not tested at all, we should at least check that the API hasn't changed :sweat_smile: 

## How to test

Run the diagnostic in CLI, and we should also verify that the check when arriving on the webadmin home page still works

## Validation
